abnf/odata-abnf-construction-rules.txt

```diff
@@ -909,12 +909,12 @@ primitiveLiteral = nullValue                  ; plain values up to int64Value
                  / geometryPoint
                  / geometryPolygon
 
-; in Atom and JSON message bodies and CSDL DefaultValue attributes
-primitiveValue = booleanValue
-               / guidValue
+; in CSDL XML DefaultValue attributes
+primitiveValue = boolean

```

Comment:
```suggestion
primitiveNonStringValue = boolean
```
If this goes out with 4.02 only, suggest renaming to `primitiveNonStringValue`, also in OData-CSDL-XML. Otherwise the analogy with `primitiveLiteral` is very misleading.

Reply:
How about making that a separate Jira issue as it requires changes to CSDL XML unrelated to [ODATA-1540](https://issues.oasis-open.org/browse/ODATA-1540)?